### PR TITLE
Update to 1.3.2 and rerender

### PR DESCRIPTION
**Pull request by gokceneraslan**
- Updated to 1.3.2
- Build number reset to 0
- toolchain builddep removed
- Rerendered

### CODE REVIEW

**Comment by jakirkham** [comment]
So I'd stick to `protobuf` on all platforms and drop `libprotobuf`. In `conda-forge`, `protobuf` contains the C++ and Python components. Though we have discussed splitting them, have lost track of where that effort is.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,22 +12,19 @@ source:
...
 build:
-  number: 3
+  number: 0
   skip: True  # [win]
 
 requirements:
   build:
-    - toolchain
     - pkg-config
     - protobuf
```
Comment (jakirkham) [inline recipe/meta.yaml]: Make sure to pin these too.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,22 +12,19 @@ source:
...
-    - libprotobuf  # [linux]
     - ncurses 5.9*
     - zlib 1.2.*
     - openssl 1.0.*
     - perl
   run:
-    - libprotobuf  # [linux]
-    - protobuf  # [osx]
+    - protobuf
```
Comment (jakirkham) [inline recipe/meta.yaml]: Same here.

File: recipe/meta.yaml (hunk 1 above)
Comment (gokceneraslan) [reply recipe/meta.yaml]: to `protobuf 3.4*` ?

Comment (jakirkham) [reply recipe/meta.yaml]: Yes `protobuf 3.4.*`.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,22 +12,19 @@ source:
...
 
 requirements:
   build:
-    - toolchain
     - pkg-config
-    - protobuf
-    - libprotobuf  # [linux]
+    - protobuf 3.4.*
     - ncurses 5.9*
     - zlib 1.2.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: Sorry. Missed this. It should be [`zlib 1.2.11`]( https://github.com/conda-forge/conda-forge.github.io/blob/06e20200a735ca2a22848caa6f6ad73c0f8f7419/scripts/pin_the_slow_way.py#L96 ).

File: recipe/meta.yaml [hunk 4]
```diff
@@ -12,22 +12,19 @@ source:
...
     - openssl 1.0.*
     - perl
   run:
-    - libprotobuf  # [linux]
-    - protobuf  # [osx]
+    - protobuf 3.4.*
     - ncurses 5.9*
     - zlib 1.2.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: Same here.

**Comment by gokceneraslan** [comment]
Now it fails on my mac:

image

File: recipe/meta.yaml [hunk 5]
```diff
@@ -12,24 +12,21 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
   skip: True  # [win]
 
 requirements:
   build:
-    - toolchain
```
Comment (jakirkham) [inline recipe/meta.yaml]: This should not be dropped.

**Comment by gokceneraslan** [comment]
What is the status? Shall I merge this now?

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
Yeah I think it’s probably fine. Would wait for the rerender to come through though.

**Comment by gokceneraslan** [comment]
Did it come through? I don't see a commit by conda-forge-admin.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
Yeah actually I don't think there is a re-rendering change needed here. Sorry for the hold up. Feel free to merge.

**Pull request merged**